docker/linux/android/Dockerfile

```diff
@@ -17,15 +17,23 @@ FROM ${FROM_IMAGE:-cobalt-build-base}
 
 # TODO: b/309157124 - add --no-install-recommends.
 RUN apt update -qqy
-RUN apt -y install wget curl
-RUN wget https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.deb
 RUN apt install -qqy \
+        wget \
```

Comment:
We don't need to install both curl and wget, one of them should be plenty :)

I would prefer if you used curl as that is what we use in the other locations but that's not a blocker for me. From GN example:
```
curl --location --silent --output /tmp/gn.zip \
    "https://chrome-infra-packages.appspot.com/dl/gn/gn/linux-amd64/+/${GN_HASH}" \
```

Reply:
Since the checks are green and this is necessary to fix the builds, I'm going to merge this now and submit a fix later.